pom.xml

```diff
@@ -2366,6 +2366,9 @@
           <family>mac</family>
           <arch>aarch64</arch>
         </os>
+        <property>
+          <name>spark2.4</name>
```

Comment:
Does the `m1-mac` profile only work for Spark 2.4 profile for the docker demo?

Reply:
I don't understand the question.
Looking at the instructions: https://hudi.apache.org/docs/docker_demo I don't think they are correct because we changed the default build profile to 3.2 and then 3.5 so I think the spark and scala versions have been needed to be specified for while

Reply:
Got it.  So by default, `m1-mac` profile should only be used by Spark 2.4, which makes sense for the docker demo.